src/Fantomas/CodePrinter.fs

```diff
@@ -2151,8 +2154,10 @@ and genMemberSig astContext node =
             | [] -> sepColon
             | _ -> sepColonWithSpacesFixed
 
+        let ziggy = mf
```

Comment:
Think you didn't mean to keep this here

Reply:
Good catch, I tend to add stuff like that to have a better debug experience in Rider.
Shouldn't be in there at all ;)